src/walletConnect/connector.ts

```diff
@@ -0,0 +1,15 @@
+import { WalletConnect } from '.'
+
+let connector: WalletConnect
+
+export function isWC() {
```

Comment:
Why not using WalletConnect here?
the other methods already do:
```suggestion
export function isWalletConnect() {
```